### DISCUSSION TITLE
Use of external database client

### DISCUSSION THREAD

**Discussion by phifa** [rtCamp/Frappe-Manager#246] [Q&A]
how could I connect to the database of a bench with Table Plus for example? I guess the mariadb container would have to map port to localhost? What is the best way to configure this?

**Comment by crelocks** [comment]
@phifa did you get this resolved?

**Comment by Xieyt** [comment]
Please refer to [this](https://github.com/rtCamp/Frappe-Manager/wiki/using-external-database) wiki guide
